superset-frontend/packages/superset-ui-core/src/query/DatasourceKey.ts

```diff
@@ -29,6 +29,7 @@ export default class DatasourceKey {
     this.id = parseInt(idStr, 10);
     this.type =
       typeStr === 'table' ? DatasourceType.Table : DatasourceType.Druid;
```

Comment:
is this druid reference still in master? cc @AAfghahi

Reply:
bump @AAfghahi or @hughhhh

Reply:
yea it's still there going to remove it now

Reply:
Hi! Sorry, I just saw this, and yes it is still in master. We should change it here.